microprofile/5.0/config/3.0/22.0.0.1-TCKResults-java11.adoc

```diff
@@ -0,0 +1,468 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Config 3.0.1.
+
+== Open Liberty 22.0.0.1 - MicroProfile Config 3.0.1 Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1/openliberty-runtime-22.0.0.1.zip[Open Liberty 22.0.0.1]
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-config-3.0.1/microprofile-config-3.0.1.html[MicroProfile Config 3.0.1]
+
+* Public URL of TCK Results Summary:
++
+link:22.0.0.1-TCKResults.html[TCK results summary]
+
+* Java runtime used to run the implementation:
++
+Java 11: JRE 11 Linux amd64-64-Bit Compressed References 20211022_282 (JIT enabled, AOT enabled); OpenJ9 based on jdk-11.0.13+8
```

Comment:
It would be nice if this used a more readable String, e.g. `IBM Semeru Runtime Open Edition (11.0.13+8)`, which is what is in the top of the messages.log file